src/effects/native/echoeffect.cpp

```diff
@@ -42,6 +42,7 @@ EffectManifest EchoEffect::getManifest() {
     time->setSemanticHint(EffectManifestParameter::SEMANTIC_UNKNOWN);
     time->setUnitsHint(EffectManifestParameter::UNITS_TIME);
     time->setDefaultLinkType(EffectManifestParameter::LINK_LINKED);
+    time->setDefaultLinkInversion(EffectManifestParameter::LINK_INVERTED);
```

Comment:
I think it makes more sense to set the feedback as the super parameter (without inversion) and put the default delay to something more sensible like 1s. Ideally you should have the delay synced to the bpm by default.